director/sensor/sensor.go

```diff
@@ -85,8 +85,10 @@ func (t sensorAggregate) Get(g cmds.Get) (messages.Event, *info.Failure, error)
 		return nil, nil, err
 	} else if sensor == nil {
 		return nil, &info.Failure{
-			Message: messages.NewMessage("failure.get.sensor", g.MessageID, g.GroupID),
-			Error:   fmt.Sprintf("%s not found.", g.ID),
+			Identifier: messages.Identifier{
+				Message: messages.NewMessage("failure.get.sensor", g.MessageID, g.GroupID),
+			},
+			Error: fmt.Sprintf("%s not found.", g.ID),
```

Comment:
Two things:

- Why not using the ID of the request inside the Identifier field?
- Why are you giving the NewMessage the old MessageID. I thought the message ID should be unique and via the GroupID we would be able to identify messages that belong together